client/templates/admin/postEditor/postEditor.html

```diff
@@ -37,8 +37,7 @@
             <textarea
               id="content-input"
               class="article-description"
-              name="content"
-              placeholder="Enter post body content">{{ model.content }}</textarea>
+              name="content">{{ model.content }}</textarea>
```

Comment:
I think this way of adding the content won't work. It will add the html as it where text  and we do want the DOM to insert the html elements stored in the content. I'm adding a commit to make this work @dagvonkr